include/nuttx/video/fb.h

```diff
@@ -283,6 +283,10 @@
 #define FBIOGET_FRAMERATE     _FBIOC(0x0015)  /* Get frame rate
                                                * Argument:            int* */
 
+#define FBIOPAN_DISPLAY       _FBIOC(0x0020)  /* Pan display
```

Comment:
```suggestion
#define FBIOPAN_DISPLAY       _FBIOC(0x0016)  /* Pan display
```
Why not follow the sequence?

Reply:
New interfaces for LCDS may be added in the future. So there's some space left over